logstash-core/src/main/java/org/logstash/common/io/DeadLetterQueueWriter.java

```diff
@@ -114,7 +112,7 @@ public DeadLetterQueueWriter(final Path queuePath, final long maxSegmentSize, fi
         this.currentQueueSize.add(getStartupQueueSize());
 
         cleanupTempFiles();
-        currentSegmentIndex = getSegmentPaths(queuePath)
+        currentSegmentIndex = DeadLetterQueueUtils.listSegmentPaths(queuePath)
```

Comment:
Maybe `import static org.logstash.common.io.DeadLetterQueueUtils.listSegmentPaths;` and then

```suggestion
        currentSegmentIndex = listSegmentPaths(queuePath)
```